sdk/ownership/backend.go

```diff
@@ -14,20 +14,25 @@ import (
 
 const backendName = "ownership"
 
+const storeIndexRessourceKey = backendName + ".index.resourceHash"
+
 // Backend is the ownership backend.
 type Backend struct {
-	storeKey *cosmostypes.KVStoreKey
+	storeKey               *cosmostypes.KVStoreKey
+	storeIndexRessourceKey *cosmostypes.KVStoreKey
 }
 
 // NewBackend returns the backend of the ownership sdk.
 func NewBackend(appFactory *cosmos.AppFactory) *Backend {
 	backend := &Backend{
-		storeKey: cosmostypes.NewKVStoreKey(backendName),
+		storeKey:               cosmostypes.NewKVStoreKey(backendName),
+		storeIndexRessourceKey: cosmostypes.NewKVStoreKey(storeIndexRessourceKey),
```

Comment:
```suggestion
		storeIndexRessourceKey: cosmostypes.NewKVStoreKey(backendName + ".index.resourceHash"),
```

Reply:
it's confusing to have 2 variable with the same name. the actual string of the key is only use once.